### PR TITLE
Jormun: Admin to admin in experimental

### PR DESCRIPTION
**Pull request by antoine-de**
implement the admin to admin journeys in experimental.

we can start(finish) from(to) either 
 * the stop points of the admin main stop areas at a cost of 0 
 * or we can walk from the center of the admin

A crow fly section is created from the admin to the stop points

linked to http://jira.canaltp.fr/browse/ITI-279

The tests will be done in artemis

TODO:
- [x] wait until some artemis tests are done locally

### CODE REVIEW

File: source/jormungandr/jormungandr/scenarios/experimental.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 from __future__ import absolute_import, print_function, unicode_literals, division
 import logging
 from jormungandr.scenarios import new_default
+from jormungandr.scenarios.default import is_admin
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/scenarios/experimental.py]: useless?

File: source/type/pb_converter.cpp [hunk 2]
```diff
@@ -429,6 +429,14 @@ void PbCreator::Filler::fill_pb_object(const ng::Admin* adm, pbnavitia::Administ
...
         admin->set_insee(adm->insee);
     }
+    for (const auto& sa: adm->main_stop_areas) {
+        auto* pb_sa = admin->add_main_stop_areas();
+        fill_pb_object(sa, pb_sa);
+        for (const auto& sp: sa->stop_point_list) {
+            auto* pb_sp = pb_sa->add_stop_points();
+            fill_pb_object(sp, pb_sp);
+        }
+    }
```
Comment (TeXitoi) [inline source/type/pb_converter.cpp:443]: It can be great if we limit the size of that to the bare minimum. depth=0 for the 2 I think.

**Review by TeXitoi — COMMENTED** [review]
Great!

File: source/type/pb_converter.cpp (hunk 2 above)
Comment (antoine-de) [reply source/type/pb_converter.cpp:443]: yes you're right!

File: source/jormungandr/jormungandr/scenarios/experimental.py [hunk 3]
```diff
@@ -97,8 +97,25 @@ def create_parameters(request):
                              forbidden_uris=request['forbidden_uris[]'])
 
 
-def _update_crowfly_duration(instance, mode, stop_area_uri, requested_point):
-    stop_points = instance.georef.get_stop_points_for_stop_area(stop_area_uri)
+def _update_crowfly_duration(instance, mode, requested_point):
```
Comment (pbougue) [inline source/jormungandr/jormungandr/scenarios/experimental.py]: `requested_place` maybe

File: source/jormungandr/jormungandr/scenarios/experimental.py [hunk 4]
```diff
@@ -118,7 +135,7 @@ def _update_crowfly_duration(instance, mode, stop_area_uri, requested_point):
         for stop_point in odt_sps:
             fallback_list[mode][stop_point.uri] = 0
             odt_stop_points.add(stop_point.uri)
-    
+
     return crowfly_sps, odt_stop_points, fallback_list
```
Comment (pbougue) [inline source/jormungandr/jormungandr/scenarios/experimental.py:139]: @TeXitoi this is always free, so I see no point in `fallback_list` here

**Review by pbougue — APPROVED** [review]
Adding tests would be nice, but I can't imagine interesting ones

File: source/jormungandr/jormungandr/scenarios/experimental.py (hunk 4 above)
Comment (TeXitoi) [reply source/jormungandr/jormungandr/scenarios/experimental.py:139]: I don't understand your remark.

File: source/jormungandr/jormungandr/scenarios/experimental.py (hunk 3 above)
Comment (TeXitoi) [reply source/jormungandr/jormungandr/scenarios/experimental.py]: I suspect that's `requested_(entry_)point`

File: source/jormungandr/jormungandr/scenarios/experimental.py (hunk 4 above)
Comment (pbougue) [reply source/jormungandr/jormungandr/scenarios/experimental.py:139]: The time is always 0 in fallback

Comment (TeXitoi) [reply source/jormungandr/jormungandr/scenarios/experimental.py:139]: https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/scenarios/experimental.py#L502

That's the same format as `g.*_places_crowfly`

Comment (pbougue) [reply source/jormungandr/jormungandr/scenarios/experimental.py:139]: ok, that's fine as it is

**Comment by antoine-de** [comment]
artemis tests done locally

**Review by TeXitoi — APPROVED** [review]

**Pull request merged**